### PR TITLE
use macros from the log crate, issue #151

### PR DESCRIPTION
**Pull request by gambhiro**
This wasn't too hard :)

Please comment if there is more to it.

### CODE REVIEW

File: src/book/mod.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ impl MDBook {
     pub fn new(root: &Path) -> MDBook {
 
         if !root.exists() || !root.is_dir() {
-            output!("{:?} No directory with that name", root);
+            info!("{:?} No directory with that name", root);
```
Comment (azerupi) [inline src/book/mod.rs]: This one could be changed to `warn`

File: src/renderer/html_handlebars/helpers/playpen.rs [hunk 2]
```diff
@@ -20,7 +20,7 @@ pub fn render_playpen(s: &str, path: &Path) -> String {
 
         // Check if the file exists
         if !playpen.rust_file.exists() || !playpen.rust_file.is_file() {
-            output!("[-] No file exists for {{{{#playpen }}}}\n    {}", playpen.rust_file.to_str().unwrap());
+            info!("[-] No file exists for {{{{#playpen }}}}\n    {}", playpen.rust_file.to_str().unwrap());
```
Comment (azerupi) [inline src/renderer/html_handlebars/helpers/playpen.rs]: `warn` here would also be better

**Comment by azerupi** [comment]
Thanks for doing this!

One thing that is missing is the actual logger in the binary. We could use [env_logger/](http://rust-lang-nursery.github.io/log/env_logger/) (I'm not familiar with any others). You would have to add env_logger to the dependencies and only import it in the binary (`src/bin/mdbook.rs`), initialize it in the main and I think it would be best to set the default log level to info.

Once you have done that it looks good enough to merge :)

**Comment by azerupi** [comment]
Looks good, thanks! :+1:

**Comment by gambhiro** [comment]
It was pretty small, but thanks for the instructions. Good experience for the beginner like me.

**Pull request merged**